### PR TITLE
Subprocess options

### PR DESCRIPTION
**Pull request by djkapner**
I would like to use this to suppress render/java output in EM_aligner_python, but, want to be based on render-python master

### CODE REVIEW

File: renderapi/client.py [hunk 1]
```diff
@@ -858,4 +873,254 @@ def transformSectionClient(stack, transformId, transformClass, transformData,
...
+
+    def to_java_args(self):
+        args = []
+        for key, value in self.__dict__.items():
+            if (value is not None) and not (key == 'kwargs'):
+                args += self.get_cmd_opt(value, "--{}".format(key))
+        return self.sanitize_cmd(args)
+
+
+class FeatureRenderParameters(ArgumentParameters):
```
Comment (fcollman) [inline renderapi/client.py]: This was not actually used, and so you can't render at at lower than scale=1.0

Comment (fcollman) [reply renderapi/client.py]: err i see it's in there .. but nonetheless this class isn't hooked up to anything.  I think it should go or get incorporated.

Comment (RussTorres) [reply renderapi/client.py]: This conversation should be on the sift client PR, I think.  Things got a little messy with commits on my end but O figured that would get in pretty soon. 
 We can take it out and re-add it when we implement something that uses it....

Comment (RussTorres) [reply renderapi/client.py]: This conversation should be on the sift client PR, I think.  Things got a little messy with commits on my end but O figured that would get in pretty soon. 
 We can take it out and re-add it when we implement something that uses it....

Comment (fcollman) [reply renderapi/client.py]: ok this PR looks cleaner now after the point match merge

**Pull request merged**